docs/document_loader.ipynb

```diff
@@ -4,72 +4,120 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Google DATABASE\n",
-    "\n",
-    "[Google DATABASE](https://cloud.google.com/DATABASE).\n",
-    "\n",
-    "Load documents from `DATABASE`."
+    "# Bigtable\n",
+    "[Bigtable](https://cloud.google.com/bigtable) is a key-value and wide-column store, ideal for fast access to structured, semi-structured, or unstructured data.\n"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Pre-reqs"
+    "## Setting up\n",
+    "\n",
+    "To run this notebook, you will need a Google Cloud Project, a Bigtable instance, and Google credentials."
```

Comment:
Should include link to instructions at least.

Reply:
Done